packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -639,13 +640,15 @@ export const ChatScreen = () => {
                     </View>
                   }
                   ListHeaderComponent={
-                    canSendMessage ? null : <ChatUnavailable chatId={chatId} />
+                    !canSendMessage ? <ChatUnavailable chatId={chatId} /> : null
                   }
                   ListFooterComponent={
                     shouldShowEndReachedIndicator ? (
                       <ChatMessageSeparator
                         content={messages.beginningReached}
                       />
+                    ) : chat?.is_blast ? (
```

Comment:
will this conflict with `shouldShowEndreachedIndicator` at all? like what if there are a lot of messages in the blast thread - would that show the end of history indicator and not `ChatBlastAudienceDisplay`? Feels like maybe the blast one should take precedence?

Reply:
good point. I'll check on that